src/data/list/count.lean

```diff
@@ -82,6 +82,16 @@ lemma count_tail : Π (l : list α) (a : α) (h : 0 < l.length),
   l.tail.count a = l.count a - ite (a = list.nth_le l 0 h) 1 0
 | (_ :: _) a h := by { rw [count_cons], split_ifs; simp }
 
+lemma count_le_length (a : α) (l : list α) : count a l ≤ l.length :=
+begin
```

Comment:
This should be shown for `countp` as well. It should also have a one-line proof, I think something like
```
lemma countp_le_length : countp p l ≤ l.length :=
by simpa only [countp_eq_length_filter] using length_le_of_sublist (filter_sublist _)
```

Reply:
@b-mehta Thanks, done.